routes/editor.js

```diff
@@ -432,6 +476,44 @@ router.post('/page/imagebackground/id/:id', function (req, res) {
             srcphone: newItem['srcphone'],
             srcmedium: newItem['srcmedium']
         };
+        var gradientEnable = (newItem['gradientEnable'] ? true : false);
+        var gradientAnimate = (newItem['gradientAnimate']) ? true : false;
+        var gradientDirection = newItem['gradientDirection'];
+        var gradientSpeed = newItem['speed'];
+        var gradientStops = [];
+        console.log(newItem['stop']);
+        for (var stop in newItem['stop']) {
+          if (newItem['stop'][stop] !== '') {
```

Comment:
this is where the problem would happen with string vs array splitting to letters